src/components/Disclaimer/index.js

```diff
@@ -1,46 +1,43 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 
 const DisclaimerModal = () => {
-//   $("#disclaimerModal").modal({ backdrop: "static", keyboard: false });
-// $(".closeModal").click(function () {
-//   $("#disclaimerModal").modal("toggle");
-// });
+  const [show, setShow] = useState(false)
+  const [btnDisable, setBtnDisable] = useState(true)
+  const [isCheckedFirst, setCheckedFirst] = useState(false)
+  const [isCheckedSecond, setCheckedSecond] = useState(false)
 
-// // Confirm Terms and service Btn
-// $(".number").on("click", function () {
-//   if ($(".number:checked").length > 1) {
-//     $("#confirmBtn")
-//       .prop("disabled", false)
-//       .addClass("confirmBtn")
-//       .removeClass("is-disabled");
-//   } else {
-//     $("#confirmBtn")
-//       .prop("disabled", true)
-//       .removeClass("confirmBtn")
-//       .addClass("is-disabled");
-//   }
-// });
+  const handleCheckedFirst = (e) => {
```

Comment:
if this isn't used anywhere else you should move this into the hook

Reply:
Resolved